### PR TITLE
[#2938] Odd feature in sorting organization by packages

### PR DESCRIPTION
**Pull request by Engerrs**

### CODE REVIEW

File: ckan/logic/action/get.py [hunk 1]
```diff
@@ -366,8 +366,10 @@ def _group_or_org_list(context, data_dict, is_org=False):
 
         query = query.filter(model.Member.group_id == model.Group.id) \
                      .filter(model.Member.table_id == model.Package.id) \
+                     .filter(model.Member.state == 'active') \
                      .filter(model.Member.table_name == 'package') \
-                     .filter(model.Package.state == 'active')
+                     .filter(model.Package.state == 'active') \
+                     .filter(model.Package.private == 'FALSE')
```
Comment (smotornyuk) [inline ckan/logic/action/get.py]: Why don't you use real `False` instead?

**Comment by smotornyuk** [comment]
@Engerrs, it works only for anonimous user. Even if you are sorting organisations by amount of public datasets, `package_count` of every organisation came from solr, thus it's slightly ambiguous, when i see some organisation with 20 private datasets created by me(logged in user)  shown after organization with only five public datasets.

**Comment by amercader** [comment]
@Engerrs is there an issue with context for this patch?

**Comment by amercader** [comment]
@Engerrs can you fix the check that @smotornyuk mentions?

**Comment by Engerrs** [comment]
@amercader , I will fix this at weekends.

**Comment by Engerrs** [comment]
@amercader I've added thix fix.

**Comment by amercader** [comment]
Thanks @Engerrs

**Comment by Engerrs** [comment]
Hi @amercader , I've also making a fixes related to @smotornyuk last comment about sorting logic.

**Comment by amercader** [comment]
Ups, sorry about that! I've reverted the changes in master. Can you create another PR when you have all the changes needed? Thanks

**Comment by Engerrs** [comment]
@amercader , yes of course.

**Pull request merged**